tea-config-generator/.editorconfig

```diff
@@ -0,0 +1,13 @@
+[*]
```

Comment:
Do we want to merge in editorconfigs  here or just gitignore them?

Reply:
I use editorconfig, so I would like it included.